src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -215,6 +215,7 @@ def check_cli(cmd):
     except Exception as ex:  # pylint: disable=broad-except
         exceptions['load_commands'] = ex
         logger.error('Error occurred loading commands!\n')
+        raise ex
```

Comment:
This `raise ex` makes this function exist, thus `exceptions` is no longer checked and useless.